lib/URI/exasol.pm

```diff
@@ -0,0 +1,23 @@
+package URI::exasol;
+use base 'URI::_odbc';
+our $VERSION = '0.18';
+
+sub default_port { 8563 }
+
+sub _dbi_param_map {
+    my $self = shift;
+    my $host = $self->host;
+    my $port = $self->_port;
+
+    # Just return the DSN if no host or port.
+    return [ DSN => scalar $self->dbname ] unless $host || $port;
+
+    # Fetch the driver from the query params.
+    require URI::QueryParam;
+    return (
+        [ EXAHOST   => $host                        ],
```

Comment:
Don't you need there database name here?

Reply:
Exasol seems to work on the basis of the cluster = the database.. So I don't think so, no, unless I'm missing something.